lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -72,6 +72,7 @@ def date_range(self, start: AnyStr, end: AnyStr) -> list:
         date_range = []
         start_date = date_parser.parse(start)
         end_date = date_parser.parse(end)
+        assert start_date <= end_date
         first_month = start_date.replace(day=1)
         last_month = end_date + relativedelta(day=31)
         assert start_date <= end_date
```

Comment:
Oh, weird; during a somewhat messy rebase, all three of these lines got duplicated. The original intent was presumably to validate the test case's *inputs* rather than to second-guess the dateutil transmogrifications. I don't see how asserting on the results makes any sense, but I'm not particularly convinced there's any value in asserting on the inputs, either. In any case, I basically left that alone.